static-site/variables.tf

```diff
@@ -15,18 +15,10 @@ variable "domain_name" {
   description = "The full domain name being added."
 }
 
-// lambda to associate with a CloudFront distribution
-variable "origin_request_lambda_arn" {
+// new site origin id
+variable "origin_id" {
```

Comment:
Can we set a default value for this?  Seems like we'll only have one origin as things stand right now, so we could probably call it `default` and move on.